projects/igniteui-angular/src/lib/core/i18n/combo-resources.ts

```diff
@@ -1,11 +1,13 @@
 export interface IComboResourceStrings {
     igx_combo_empty_message?: string;
+    igx_combo_enabledFilter_placeholder?: string;
     igx_combo_addCustomValues_placeholder?: string;
     igx_combo_clearItems_placeholder?: string;
```

Comment:
Eh, somewhat assumed this was going to be added with this PR, but instead it went in with #14620 (and propagated back too), so a bit late for this comment:
Do be careful with naming public API - it's not a `addCustomValues_placeholder` if it's also the text of the button, no? So the name should've been more generic or reflective of the use. I do hope that's been communicated so the translation makes sense in both places, though we might very well consider 
Similar for `clearItems_placeholder`.. that's a tooltip or aria label.

---

As for the actual `enabledFilter_placeholder` - could be more descriptive, either filter field or filter search placeholder (to go with the old prop I guess).